firefox-install-demo.toml

```
@@ -0,0 +1,77 @@
+[project]
+
+name = "Firefox Installation Success Rate"
+platform = "firefox_desktop"
+xaxis = "submission_date"
+start_date = "2021-11-01"
+# We want continuous monitoring of this data
+#end_date =
+skip_default_metrics = true
+
+metrics = [
+    "success_rate",
+]
+
+[project.population]
+
+data_source = "firefox_installs"
+branches = [
+    "Win7",
+    "Win8",
+    "Win8.1",
+    "Win10"
+]
+monitor_entire_population = true
+
+
+[metrics]
+
+
+[metrics.install_success_rate]
+data_source = "firefox_installs"
+select_expression = "round(avg(if(succeeded, 1, 0)) * 100, 1)"
+type = "scalar"
+
+[metrics.install_success_rate.statistics]
+sum = {}
+mean = {}
+
+[data_sources]
+
+[data_sources.firefox_installs]
+from_expression = """
+    (
+        WITH success_rate_per_branch AS (
+            SELECT 
+                DATE(submission_timestamp) AS submission_date,
+                build_channel,
+                succeeded,
+                CASE 
+                    WHEN os_version LIKE '6.1%' THEN 'Win7'
+                    WHEN os_version LIKE '6.2%' THEN 'Win8'
+                    WHEN os_version LIKE '6.3%' THEN 'Win8.1'
+                    WHEN os_version LIKE '10%' THEN 'Win10'
+                    ELSE "other"
+                END as branch,
+            FROM firefox_installer.install
```

the project needs to be specified since this query is potentially run in a project other than mozdata where the dataset does not exist
```suggestion
            FROM mozdata.firefox_installer.install
```